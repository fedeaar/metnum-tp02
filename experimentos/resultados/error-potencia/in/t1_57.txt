3.320294280345349307e+02 -9.746833972336229834e+00 -3.003950981784698371e+01 -2.394857066113042521e+01 -7.103293505318416123e+01 -1.445372611974600119e+01 -3.488000760159955860e+01 -2.422988678622071745e+01 3.903737676121766143e+00 -7.861241162818627615e+00 -4.175062304753100051e+01 8.613693640311874233e+00 -1.306544996994550090e+01 -3.738283137661161248e+01 -4.094543245717966684e+00 -1.176934936217301608e+00 -6.207426415346748882e+00 -5.147848326433892119e+00 -1.181036321922165300e+01 -4.027297134547277224e+00
-9.746833972336229834e+00 4.043728722860798257e+02 -2.633160989741977431e+01 -2.267290184346252957e+01 -6.168556370818365053e+01 -1.556961699659674103e+01 -3.032491768934545462e+01 -2.149358585917202191e+01 2.237955631892846409e+00 -8.377176328562539354e+00 -3.758216208456899210e+01 4.292279773905221418e+00 -1.149169422503367954e+01 -3.387649387973333859e+01 -4.018549725841829456e+00 -4.090170719890684481e+00 -5.482572577287056603e+00 -4.667779014592026599e+00 -1.048584285265011395e+01 -3.619307860180272840e+00
-3.003950981784698371e+01 -2.633160989741977076e+01 5.646724904337391990e+02 -8.092318902171287220e+01 -1.788451504455445047e+02 -6.957596175874053301e+01 -8.820324070163412955e+01 -6.597742660847771390e+01 -2.839542676048459224e+00 -3.684119569552761675e+01 -1.196949870477791222e+02 -1.336701041872819395e+01 -3.449641196367914375e+01 -1.096584323866660640e+02 -1.539827544158102945e+01 -3.670021865089086077e+01 -1.664049015727018599e+01 -1.513116370313644943e+01 -3.226088125692382391e+01 -1.148103421678185931e+01
-2.394857066113042521e+01 -2.267290184346252246e+01 -8.092318902171288642e+01 -2.042890257067813593e+02 -2.162620475825451081e+02 8.570784550401431545e+01 -1.046957177393081651e+02 -5.433423770870513891e+01 6.140299205326204657e+01 4.270492351876561798e+01 -7.013552796779600840e+01 1.632455978715472611e+02 -3.352208471940000578e+01 -5.308294234701827463e+01 7.426459541487906435e+00 1.282881715869326626e+02 -1.494467511836205276e+01 -7.190571095095664944e+00 -2.608724148177953950e+01 -7.018758801969810612e+00
-7.103293505318416123e+01 -6.168556370818366474e+01 -1.788451504455445047e+02 -2.162620475825451365e+02 5.715023457805474436e+02 -2.132073200092615082e+02 -1.967149625849541223e+02 -1.554153386052453527e+02 -2.866995027945796437e+01 -1.119718947789745158e+02 -2.917578746886298404e+02 -9.157832550045775122e+01 -7.949585115839295213e+01 -2.711461908785824448e+02 -4.320592896312017217e+01 -1.411130733854818970e+02 -3.877018578019327322e+01 -3.746019864321479531e+01 -7.615889577670395738e+01 -2.788465293417536017e+01
-1.445372611974599231e+01 -1.556961699659674103e+01 -6.957596175874056144e+01 8.570784550401431545e+01 -2.132073200092615650e+02 -2.798402555354404626e+02 -1.017660551619739380e+02 -3.473915089877431939e+01 1.085074581295672260e+02 1.066597097153857021e+02 -1.394827713400507108e+01 2.936823275135147924e+02 -2.698782491524060489e+01 6.500905753764741668e+00 2.659286694733045309e+01 2.542299245591842123e+02 -1.090300684821327692e+01 1.128366168657106483e+00 -1.615714214866985188e+01 -1.840832522849117137e+00
-3.488000760159955149e+01 -3.032491768934545462e+01 -8.820324070163412955e+01 -1.046957177393081651e+02 -1.967149625849541223e+02 -1.017660551619739380e+02 8.236923013885945011e+02 -7.635110534408607919e+01 -1.275797676188026841e+01 -5.348819407174763541e+01 -1.427403939832689446e+02 -4.136279773140243776e+01 -3.916046692622806802e+01 -1.324315623148699501e+02 -2.080731001801769509e+01 -6.602546098048117074e+01 -1.907250811599598705e+01 -1.829342271491136174e+01 -3.740468346158584012e+01 -1.364822327722051121e+01
-2.422988678622071745e+01 -2.149358585917202191e+01 -6.597742660847771390e+01 -5.433423770870515312e+01 -1.554153386052453527e+02 -3.473915089877433360e+01 -7.635110534408607919e+01 3.175198547063533283e+02 7.351827120326102083e+00 -1.880036078727992788e+01 -9.271599150013840074e+01 1.555530454852817535e+01 -2.873659082932745079e+01 -8.324973192390071119e+01 -9.436352438922282815e+00 -5.742261183337012298e+00 -1.367640178172340804e+01 -1.146687052832935194e+01 -2.607733589010248920e+01 -8.937368908607814078e+00
3.903737676121763034e+00 2.237955631892849961e+00 -2.839542676048465442e+00 6.140299205326204657e+01 -2.866995027945798213e+01 1.085074581295671976e+02 -1.275797676188027552e+01 7.351827120326110077e+00 -9.087745330984666907e+02 5.556884994004172995e+01 3.337257040998517255e+01 1.242600079372200241e+02 2.413134246369826597e-01 3.844523233854268796e+01 1.588247535493053775e+01 1.157690626084662568e+02 9.800547529446026473e-01 5.399292497986547090e+00 3.933339369985884737e+00 2.995744509183684734e+00
-7.861241162818629391e+00 -8.377176328562537577e+00 -3.684119569552761675e+01 4.270492351876561798e+01 -1.119718947789745300e+02 1.066597097153856737e+02 -5.348819407174763541e+01 -1.880036078727992432e+01 5.556884994004174416e+01 -4.238914505752820219e+02 -8.955662932511682328e+00 1.503148502988088353e+02 -1.435243978235998341e+01 1.676368722307185477e+00 1.339678929957571896e+01 1.297426491972654219e+02 -5.839167199794371932e+00 3.498791534506773826e-01 -8.767802332535286070e+00 -1.116776477854015903e+00
-4.175062304753100761e+01 -3.758216208456899210e+01 -1.196949870477791222e+02 -7.013552796779599419e+01 -2.917578746886298404e+02 -1.394827713400502844e+01 -1.427403939832689446e+02 -9.271599150013840074e+01 3.337257040998517255e+01 -8.955662932511696539e+00 1.464915654292294978e+00 8.335688902701349434e+01 -5.147396594403461734e+01 -1.322802942883066351e+02 -9.852509557428970055e+00 4.134015489762931850e+01 -2.411164893109807750e+01 -1.817405356725895871e+01 -4.505355222731341769e+01 -1.470592871927359013e+01
8.613693640311876010e+00 4.292279773905224971e+00 -1.336701041872819040e+01 1.632455978715472327e+02 -9.157832550045775122e+01 2.936823275135147924e+02 -4.136279773140244487e+01 1.555530454852817890e+01 1.242600079372200241e+02 1.503148502988088069e+02 8.335688902701349434e+01 -5.629817903964757306e+02 -1.781308009019967997e+00 9.802842465634059010e+01 4.261729403291362672e+01 3.159969135759980077e+02 1.517505086997284369e+00 1.378632119932104239e+01 8.537687830556045299e+00 7.430466891949109254e+00
-1.306544996994550090e+01 -1.149169422503367954e+01 -3.449641196367914375e+01 -3.352208471940001289e+01 -7.949585115839296634e+01 -2.698782491524060845e+01 -3.916046692622806802e+01 -2.873659082932745434e+01 2.413134246369857683e-01 -1.435243978235998696e+01 -5.147396594403462444e+01 -1.781308009019966221e+00 5.378564520145133656e+02 -4.689876196903233563e+01 -6.240483660778938457e+00 -1.230916417414725572e+01 -7.276568297017267817e+00 -6.468193615921099315e+00 -1.404014492812101800e+01 -4.944093682255807032e+00
-3.738283137661161248e+01 -3.387649387973333148e+01 -1.096584323866660924e+02 -5.308294234701829595e+01 -2.711461908785825017e+02 6.500905753764748773e+00 -1.324315623148699501e+02 -8.324973192390071119e+01 3.844523233854270217e+01 1.676368722307171266e+00 -1.322802942883066351e+02 9.802842465634059010e+01 -4.689876196903232852e+01 -3.482238365077541431e+01 -6.171544043193694762e+00 5.820732249034748662e+01 -2.181492953485616937e+01 -1.561733673723955995e+01 -4.038968277076401847e+01 -1.288035609193598141e+01
-4.094543245717966684e+00 -4.018549725841829456e+00 -1.539827544158102945e+01 7.426459541487906435e+00 -4.320592896312017217e+01 2.659286694733043888e+01 -2.080731001801769864e+01 -9.436352438922281038e+00 1.588247535493053419e+01 1.339678929957571896e+01 -9.852509557428970055e+00 4.261729403291362672e+01 -6.240483660778936681e+00 -6.171544043193692985e+00 -3.310640506959919094e+02 3.525737785276563585e+01 -2.697057425464825187e+00 -8.173215114513878721e-01 -4.491294175536146405e+00 -1.019514441741519173e+00
-1.176934936217301608e+00 -4.090170719890684481e+00 -3.670021865089086788e+01 1.282881715869326626e+02 -1.411130733854818970e+02 2.542299245591842123e+02 -6.602546098048114231e+01 -5.742261183336998087e+00 1.157690626084662568e+02 1.297426491972654219e+02 4.134015489762933271e+01 3.159969135759979508e+02 -1.230916417414725217e+01 5.820732249034750083e+01 3.525737785276563585e+01 -4.226857664347111267e+02 -3.706618663277161385e+00 8.275697136127972087e+00 -1.933246183685312980e+00 3.434889510850991812e+00
-6.207426415346748882e+00 -5.482572577287056603e+00 -1.664049015727018599e+01 -1.494467511836205631e+01 -3.877018578019328032e+01 -1.090300684821328048e+01 -1.907250811599598705e+01 -1.367640178172340981e+01 9.800547529446008710e-01 -5.839167199794373708e+00 -2.411164893109807750e+01 1.517505086997286146e+00 -7.276568297017266040e+00 -2.181492953485616937e+01 -2.697057425464826075e+00 -3.706618663277163161e+00 4.535200726090699845e+02 -3.006823104912002886e+00 -6.675504556517151578e+00 -2.319939263436206822e+00
-5.147848326433891231e+00 -4.667779014592026599e+00 -1.513116370313644765e+01 -7.190571095095669385e+00 -3.746019864321478821e+01 1.128366168657104707e+00 -1.829342271491135818e+01 -1.146687052832935017e+01 5.399292497986546202e+00 3.498791534506753287e-01 -1.817405356725896226e+01 1.378632119932104239e+01 -6.468193615921099315e+00 -1.561733673723956173e+01 -8.173215114513880941e-01 8.275697136127968534e+00 -3.006823104912002886e+00 7.085744255588465990e+01 -5.562517735641112182e+00 -1.770180298957579224e+00
-1.181036321922165300e+01 -1.048584285265011395e+01 -3.226088125692382391e+01 -2.608724148177953595e+01 -7.615889577670395738e+01 -1.615714214866984477e+01 -3.740468346158584723e+01 -2.607733589010248920e+01 3.933339369985884737e+00 -8.767802332535287846e+00 -4.505355222731341769e+01 8.537687830556045299e+00 -1.404014492812101622e+01 -4.038968277076402558e+01 -4.491294175536147293e+00 -1.933246183685312980e+00 -6.675504556517151578e+00 -5.562517735641112182e+00 3.422871149184729802e+02 -4.344609951898547706e+00
-4.027297134547277224e+00 -3.619307860180272840e+00 -1.148103421678185931e+01 -7.018758801969811500e+00 -2.788465293417535662e+01 -1.840832522849118025e+00 -1.364822327722051121e+01 -8.937368908607814078e+00 2.995744509183684734e+00 -1.116776477854016347e+00 -1.470592871927359191e+01 7.430466891949108366e+00 -4.944093682255806144e+00 -1.288035609193598141e+01 -1.019514441741519617e+00 3.434889510850990035e+00 -2.319939263436206822e+00 -1.770180298957579224e+00 -4.344609951898547706e+00 1.715739657118413959e+02
