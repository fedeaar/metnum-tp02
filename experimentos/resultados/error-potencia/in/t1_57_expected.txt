9.690000000000000000e+02
-9.540000000000000000e+02
9.210000000000000000e+02
-9.040000000000000000e+02
-7.080000000000000000e+02
6.430000000000000000e+02
5.530000000000000000e+02
-4.900000000000000000e+02
-4.780000000000000000e+02
4.570000000000000000e+02
4.130000000000000000e+02
3.710000000000000000e+02
3.550000000000000000e+02
3.430000000000000000e+02
-3.340000000000000000e+02
-2.120000000000000000e+02
1.730000000000000000e+02
1.530000000000000000e+02
7.900000000000000000e+01
7.300000000000000000e+01
